cogs/Staffrequirements.py

```diff
@@ -0,0 +1,89 @@
+"""Allow to quickly check a user account for create and join time."""
+# import re
+import time
+from datetime import datetime
+
+import discord
+from discord.commands import option
+from discord.ext import commands
+
+from util.EmbedBuilder import EmbedBuilder
+from util.Logging import log
+
+
+class Staffrequirement(commands.Cog):
+    """Check key account statistics of a staff applicant's account."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+
+    # Allows the user to set the keeptime to a value other than the default
+    @commands.slash_command(
+        name="check-requirements",
+        description="Check if the account meets requirements for staff.",
+    )
+    # @option(
+    #    name="user ID",
+    #    description="The uID of the account to check.",
+    #    #autocomplete=get_keywords,
+    # )
+    async def add_alert(self, ctx: commands.Context) -> None:
```

Comment:
Copy pasted  function name from other cog?

Reply:
Rebased in https://github.com/Arborym/Pax-Academia/pull/58/commits/2700433aa6c2ddfa39d5dfbb320bb23a620ff985